src/main/java/roomescape/domain/reservationTime/service/ReservationTimeService.java

```diff
@@ -0,0 +1,49 @@
+package roomescape.domain.reservationTime.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.domain.reservationTime.domain.ReservationTime;
+import roomescape.domain.reservationTime.dto.ReservationTimeRequest;
+import roomescape.domain.reservationTime.dto.ReservationTimeResponse;
+import roomescape.domain.reservationTime.repository.ReservationTimeRepository;
+import roomescape.global.exception.RoomescapeBadRequestException;
+import roomescape.global.exception.RoomescapeServerException;
+
+@Service
+public class ReservationTimeService {
+
+    private final ReservationTimeRepository reservationTimeRepository;
+
+    public ReservationTimeService(final ReservationTimeRepository reservationTimeRepository) {
+        this.reservationTimeRepository = reservationTimeRepository;
+    }
+
+    @Transactional
+    public ReservationTimeResponse createReservationTime(final ReservationTimeRequest timeRequest) {
+        ReservationTime newReservationTime = reservationTimeRepository.create(timeRequest.toReservationTime());
+
+        return ReservationTimeResponse.fromReservationTime(newReservationTime);
+    }
+
+    @Transactional(readOnly = true)
+    public List<ReservationTimeResponse> getReservationTimes() {
+
+        return reservationTimeRepository.findAll()
+                .stream()
+                .map(ReservationTimeResponse::fromReservationTime)
+                .collect(Collectors.toList());
```

Comment:
JDK 16부터는 아래와 같이 사용이 가능해요!

```suggestion
                .toList();
```

Reply:
오 ! 감사합니다 
반영했어요 ~~

Reply:
여전히 반영이 안된 부분이 있어보이네요~~ 👀 
mac + intelliJ 를 사용한다면 `cmd + shift + R` 단축키로 변경해야 하는 부분을 모두 탐색하거나 일괄 변경할 수 있답니다~

Reply:
아 ㅠㅠ 명령어 잘 사용할게요 !!

안된 부분도 반영했습니다 ~!